yarn.lock

```diff
@@ -40,7 +40,14 @@
     "@nicolo-ribaudo/chokidar-2" "2.1.8-no-fsevents.3"
     chokidar "^3.4.0"
 
-"@babel/code-frame@7.12.11", "@babel/code-frame@^7.0.0", "@babel/code-frame@^7.12.11", "@babel/code-frame@^7.18.6":
+"@babel/code-frame@7.12.11":
```

Comment:
I wonder why `yarn.lock` changed so much... or changed at all. We already had `ansi-styles` in there (several versions, in fact!)

```
$ yarn why ansi-styles
yarn why v1.22.19
[1/4] Why do we have the module "ansi-styles"...?
[2/4] Initialising dependency graph...
[3/4] Finding dependency...
[4/4] Calculating file sizes...
=> Found "ansi-styles@4.3.0"
info Has been hoisted to "ansi-styles"
info Reasons this module exists
   - "workspace-aggregator-bc9a3b66-47ad-4edf-bc7e-1dd29cadd912" depends on it
   - Hoisted from "_project_#slice-ansi#ansi-styles"
   - Hoisted from "_project_#wrap-ansi#ansi-styles"
   - Hoisted from "_project_#cli-truncate#slice-ansi#ansi-styles"
   - Hoisted from "_project_#eslint#chalk#ansi-styles"
   - Hoisted from "_project_#log-symbols#chalk#ansi-styles"
   - Hoisted from "_project_#inquirer#chalk#ansi-styles"
   - Hoisted from "_project_#@agoric#web-components#lint-staged#chalk#ansi-styles"
   - Hoisted from "_project_#inquirer#ora#chalk#ansi-styles"
   - Hoisted from "_project_#@agoric#ertp#tsd#eslint-formatter-pretty#chalk#ansi-styles"
   - Hoisted from "_project_#@agoric#web-components#@web#test-runner#@web#test-runner-core#log-update#wrap-ansi#ansi-styles"
info Disk size without dependencies: "84KB"
info Disk size with unique dependencies: "168KB"
info Disk size with transitive dependencies: "188KB"
info Number of shared dependencies: 2
=> Found "chalk#ansi-styles@3.2.1"
info This module exists because "_project_#chalk" depends on it.
info Disk size without dependencies: "16KB"
info Disk size with unique dependencies: "100KB"
info Disk size with transitive dependencies: "120KB"
info Number of shared dependencies: 2
=> Found "npm-run-all#ansi-styles@3.2.1"
info This module exists because "_project_#npm-run-all" depends on it.
info Disk size without dependencies: "16KB"
info Disk size with unique dependencies: "100KB"
info Disk size with transitive dependencies: "120KB"
info Number of shared dependencies: 2
=> Found "ava#ansi-styles@6.2.1"
info Reasons this module exists
   - "_project_#ava" depends on it
   - Hoisted from "_project_#ava#cli-truncate#slice-ansi#ansi-styles"
info Disk size without dependencies: "32KB"
info Disk size with unique dependencies: "32KB"
info Disk size with transitive dependencies: "32KB"
info Number of shared dependencies: 0
=> Found "cliui#ansi-styles@3.2.1"
info Reasons this module exists
   - "_project_#cliui#wrap-ansi" depends on it
   - Hoisted from "_project_#cliui#wrap-ansi#ansi-styles"
info Disk size without dependencies: "16KB"
info Disk size with unique dependencies: "100KB"
info Disk size with transitive dependencies: "120KB"
info Number of shared dependencies: 2
Done in 0.56s.
```

Reply:
Yeah, I don't know why it changed at all. Probably a side effect of `yarn add`. I expect it'll happen the next time someone does `yarn add` so I'll leave it in.

Reply:
Hmm, it's failing in CI because the lock is changing again. I reverted the yarn.lock changes and while I was debugging I noticed that chalk could be safely upgraded in the packages that depend on it, so I did that too.